changelog/28126.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+auto-auth/cert: support watching changes on certificate/key files and notifying the auth handler when `enable_reauth_on_new_credentials` is enabled.
+```
```

Comment:
We should add a new changelog here for adding `enable_reauth_on_new_credentials`. I can't give a great example because GitHub gets confused by the formatting, but you can add two changelog blocks in the same file (see `changelog/24790.txt` as an example.

The second improvement should say something like
`auto-auth: support new config option, `enable_reauth_on_new_credentials`, supporting more frequent authentication on certain auto-auth types`

Reply:
Hmmm add a new one in the note, pls have a look if it's in the right format. Although `certain` here is a bit confusing to the user probably

Reply:
This looks good, thanks!